linode/ipv6range/tmpl/template.go

```diff
@@ -48,11 +49,12 @@ func ReassignmentStep2(t *testing.T, label, region string) string {
 		})
 }
 
-func RaceCondition(t *testing.T, label, region string) string {
+func RaceCondition(t *testing.T, label, region1, region2 string) string {
```

Comment:
For context, this test was created to test a race condition in the API that would occur between multiple IPv6 ranges created in the same DC due to an improperly scoped lock (only locking on Linode ID rather than Linode ID + DC). 

If possible, could we try to revert to a single region for the test template the ensure there are no regressions?

Reply:
I see. How to make this test case work locally? Due to the API limitation I can't create multiple ipv6 ranges in the same region when running it. Should I submit a request to the API team?